pypeit/slittrace.py

```diff
@@ -1293,6 +1304,49 @@ def get_maskdef_offset(self, sobjs, slits_left, platescale, slitmask_off, bright
                       f'Slitmask offset cannot be estimated in {self.detname}.')
             self.maskdef_offset = 0.0
 
+        return
+
+    def get_maskdef_extract_fwhm(self, sobjs, platescale, fwhm_parset):
+        """
```

Comment:
Provide a brief description of the method

Reply:
Done